modules/mgmt-installing-the-node-maintenance-operator.adoc

```diff
@@ -0,0 +1,193 @@
+// Module included in the following assemblies:
+//
+// virt/node_maintenance/mgmt-node-maintenance.adoc
+
+[id="installing-the-node-maintenance-operator_{context}"]
+= Installing the Node Maintenance Operator
```

Comment:
Oh... while technically all this looks correct on a quick look, I don't think this belongs into the OCP documentation. You explain here how to install **_upstream_** NMO. But these docs are about **_downstream_**. And downstream NMO is installed automatically when you install Openshift Virtualization (until further notice).

Reply:
Ah OK, I don't think I understood how this worked then - so this whole section can just be removed?

Reply:
yes, both CLI and UI installation